index.js

```diff
@@ -158,8 +158,9 @@ NYC.prototype._prepGlobPatterns = function (patterns) {
 
 NYC.prototype.addFile = function (filename) {
   var relFile = path.relative(this.cwd, filename)
-  var source = stripBom(fs.readFileSync(filename, 'utf8'))
+  var source = this._readTranspiledSource(path.resolve(this.cwd, filename))
   var instrumentedSource = this._maybeInstrumentSource(source, filename, relFile)
```

Comment:
What if we've already wrapped require? Then the result of `_readTranspiledSource` will already be instrumented. Is `addFile` only called for reports?